<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderPage</class>
 <widget class="QWidget" name="OrderPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>422</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>我の订单</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background: #7f7fd5;  /* fallback for old browsers */
background:qlineargradient(spread:pad,x1:1,y1:0,x2:0,y2:0,stop:0 #91eae4,stop:1 #7f7fd5);
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>11</number>
      </property>
      <property name="rightMargin">
       <number>11</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Adobe 黑体 Std R&quot;;
background:transparent;</string>
        </property>
        <property name="text">
         <string>订单号:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lab_order">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;楷体&quot;;
background:transparent;</string>
        </property>
        <property name="text">
         <string>订单号</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lab_status">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;微软雅黑&quot;;
color:rgb(234, 78, 0);
background:transparent;</string>
        </property>
        <property name="text">
         <string>订单状态</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_back">
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color:rgb(101, 202, 0);
border-style:outset;                 
border-width:4px;                  
border-radius:10px;
border-color:rgba(255,255,255,30);
font: 11pt &quot;华光准圆_CNKI&quot;;
color:white;
padding:6px;
}
QPushButton:pressed{
background-color:rgba(100,255,100,200);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);
}

QPushButton:hover{
background-color:rgba(100,255,100,100);
border-color:rgba(255,255,255,200);
color:rgba(0,0,0,200);
}
</string>
        </property>
        <property name="text">
         <string>上一个</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_next">
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color:rgb(101, 202, 0);
border-style:outset;                 
border-width:4px;                  
border-radius:10px;
border-color:rgba(255,255,255,30);
font: 11pt &quot;华光准圆_CNKI&quot;;
color:white;
padding:6px;
}
QPushButton:pressed{
background-color:rgba(100,255,100,200);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);
}

QPushButton:hover{
background-color:rgba(100,255,100,100);
border-color:rgba(255,255,255,200);
color:rgba(0,0,0,200);
}
</string>
        </property>
        <property name="text">
         <string>下一个</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Adobe 黑体 Std R&quot;;</string>
        </property>
        <property name="text">
         <string>下单时间:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lab_time">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;华光魏体_CNKI&quot;;
color:rgb(0, 170, 255);</string>
        </property>
        <property name="text">
         <string>time</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Adobe 黑体 Std R&quot;;</string>
        </property>
        <property name="text">
         <string>总金额$</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lab_price">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;华光魏体_CNKI&quot;;
color:rgb(255, 0, 127);</string>
        </property>
        <property name="text">
         <string>order_price</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
